source/dokku.rst

```diff
@@ -0,0 +1,222 @@
+Deploying to a Dokku server
+===========================
+
+Projects created using a recent version of the Caktus
+`django-project-template <https://github.com/caktus/django-project-template>`_
+can be deployed to a Dokku server as follows.
+
+These instructions assume
+
+* that ``ssh dokku`` will connect you to your dokku server as the dokku user
+* the `Postgres plugin <https://github.com/dokku/dokku-postgres>`_
+  is installed on the dokku server
+* the `Lets Encrypt plugin <https://github.com/dokku/dokku-letsencrypt>`_
+  is installed on the dokku server
+* an admin has created a directory on the dokku server named
+  ``/var/lib/dokku/data/storage/{{ project_name }}`` that we can use for
+  storage that persists across deploys, and that is writable by the
+  ``dokku`` user.
+
+Running::
+
+     ./dokku_first_deploy.sh
+
+from the new project directory will do much of the following. Or you
+can go through step-by-step yourself as follows.
+
+Create a new app::
+
+    $ ssh dokku apps:create {{ project_name }}
+    Creating {{ project_name }}... done
+
+Mount the persistent storage at ``/storage`` inside the container,
+and set a config variable to tell Django where we're going to store
+our media::
+
+    $ ssh dokku storage:mount {{ project_name }} /var/lib/dokku/data/storage/{{ project_name }}:/storage
+    $ ssh dokku config:set {{ project_name }} MEDIA_ROOT=/storage/media MEDIA_URL=/media
```

Comment:
will this actually create the host-resident directory and establish the right permissions, or does that have to be manually?

Reply:
Has to be done by someone with admin privs on the server.

Reply:
I think that should be added as a note then, someone just following these steps will not be able to complete them as written without it.